.circleci/config.yml

```diff
@@ -26,11 +26,13 @@ workflows:
   test-all-node-versions:
     jobs:
       - test:
-          docker_image: circleci/node:10-browsers
+          docker_image: cimg/node:12.13
```

Comment:
Why did you decide to use those specific minor versions instead of only major ones as in legacy `circleci/node` images?

Tests for `node:12.13` are failing because of `import` statement. 
See the details [here](https://app.circleci.com/pipelines/github/iLib-js/ilib-address/11/workflows/7bc43fce-ec1a-4713-81d0-81d73d71a3b3/jobs/54?invite=true#step-103-641_57).

Reply:
Node.js `v12.13` does not fully support ES modules without using the `--experimental-modules flag` or specific configurations in project.
In Node.js v12.17.0 and later ES module support is more stable.